src/sentry/models/sentryapp.py

```diff
@@ -147,7 +128,9 @@ def save(self, *args, **kwargs):
         return super(SentryApp, self).save(*args, **kwargs)
 
     def is_installed_on(self, organization):
-        return self.organizations.filter(pk=organization.pk).exists()
+        from sentry.models import SentryAppInstallation
```

Comment:
I wonder if you wouldn't need to do this if you used `from sentry.models.sentryappinstallation import SentryAllInstallation`, could be worth trying

Reply:
@wedamija good call, this works